purview/api/v1.py

```diff
@@ -23,15 +24,34 @@ def about():
     return jsonify({'version': version})
 
 
+def get_all_relationships(item):
+    all_relations = dict()
+    for model in all_models:
```

Comment:
Shouldn't we determine what models the item has a relationship to rather than cycle through all of them?

Reply:
I was looking into this and an alternative approach would involve getting all relations of an item using item.__all_relationships__. Then traversing through the list of tuples returned and extracting unique model names. Then finding all nodes related to the item. Which I thought wouldn't be less computationally intense if not more. What are your thoughts on this?